tests/foreman/ui/test_computeresource.py

```diff
@@ -127,19 +129,25 @@ def test_create_libvirt_resource_different_names_negative(self, name):
         @Assert: A libvirt Compute Resource is not created
 
         """
+        include_list = [' ']
```

Comment:
Two reasons:
* This is a proactive approach to add more data in the future.  I am pretty sure that there can be other invalid entities that can be tested, but it is beyond the scope of this PR. 
* This is the standard template I use to add additional data points to an existing list - I used this standard across API tests.